ez_gpg/ui.py

```diff
@@ -1,17 +1,22 @@
 # vim:ff=unix ts=4 sw=4 expandtab
 
-import gi
 import os
+
+# https://github.com/pypa/setuptools
 import pkg_resources
 
+# https://wiki.gnome.org/Projects/PyGObject
+import gi 
```

Comment:
Putting a comment in is fine but both the comment is wrong (gobject-introspection API and not PyGObject)  and the standard placement of imports is at the top even though the checking of requirements is below it.